python/examples/integrations/bentoml/train.py

```diff
@@ -0,0 +1,22 @@
+import bentoml
+from sklearn import datasets, neighbors
+
+# Load training data set
+iris = datasets.load_iris()
+X, y = iris.data, iris.target
+
+# Create & train the model
+knn = neighbors.KNeighborsClassifier(n_neighbors=5)
+knn.fit(X, y)
```

Comment:
it would be great to store the training data's profile as reference, using whylogs+WhyLabs :)

Reply:
I am going to do that in the blog post! I originally had it in here too, but I felt like it wasn't actually part of integrating with BentoML. I could add it back in, or maybe we make a note saying you can make a reference profile with training data.